src/main/java/logging/Logger.java

```diff
@@ -0,0 +1,7 @@
+package logging;
+
+public interface Logger {
+    void setOutputFile(String outPutFileNameInJson);
```

Comment:
Itt Json object-et kellene hogy kapjanak, legalábbis az a lényeg, hogy lássanak egy olyan példát is ahol nem egy karaktersor utazik rajta. Tudom, hogy a Json is egy string, de itt az a lényeg, hogy lássanak egy példát a kiválasztott JSON lib-re, amit a projektekben is használni szokatok

Reply:
Nem nagyon használunk így kimondottan JSONLib-eket, mivel erről gondoskodik a keretrendszer. 
Spring bootban, amikor meghívnak egy végpontot a keretrendszer beparseolja a megfelelő osztályba (objektumba) a json-t és úgy adja tovább a dedikált controllernek.
Pl, ha valaki küld egy Muemlek-et a PUT végponton akkor az így néz ki a controllerben:
` @PutMapping("{azon}")
    public void updateMuemlek(@PathVariable int azon, @RequestBody MuemlekDTO muemlekDTO) {
        if (azon != muemlekDTO.getAzon()) {
            throw new IllegalArgumentException("Nem egyezik meg a paraméterben és a bodyban található azonosító!");
        }
        muemlekService.updateMuemlek(muemlekDTO);
    }
`
Csak nagyon speciális esetben hozunk létre JSONObject-et, pl a dummy végpontoknál, hogy ne kelljen létrehozni a tényleges objektumokat egy sima stringet convertáltunk át JSONObjectté, illetve a system teszteknél használtuk még.